light/client_test.go

```diff
@@ -905,6 +905,27 @@ func TestClient_NewClientFromTrustedStore(t *testing.T) {
 	assert.EqualValues(t, l1.Height, h.Height)
 }
 
+func TestClient_NewClientFromTrustedStore2(t *testing.T) {
+	// empty DB
+	db := dbs.New(dbm.NewMemDB(), chainID)
+
+	c, err := light.NewClientFromTrustedStore(
+		chainID,
+		trustPeriod,
+		fullNode,
+		[]provider.Provider{fullNode},
+		db,
+	)
+
+	if err != nil {
+		assert.Error(t, err)
```

Comment:
You know this only checks that `err != nil` like in the previous line, right?

Reply:
I can't assert the type because you've asked me to write a test that works for both `main` and this PR.